src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -72,6 +72,7 @@ import { buildVislibDimensions } from 'ui/visualize/loader/pipeline_helpers/buil
 import 'ui/capabilities/route_setup';
 
 import { data } from 'plugins/data/setup';
+import { brushHandler } from '../../../../metrics/public/lib/create_brush_handler';
```

Comment:
Will need to coordinate with https://github.com/elastic/kibana/pull/33558/files, where this function has been renamed and the parameters changed a bit.